ixfr.c

```diff
@@ -2419,45 +2362,41 @@ static void clear_temp_table_of_rr(struct domain_table* temptable,
 
 /* read ixfr data new SOA */
 static int ixfr_data_readnewsoa(struct ixfr_data* data, struct zone* zone,
-	FILE* in, const char* ixfrfile, struct region* tempregion,
+	struct rr *rr, zone_parser_t *parser, struct region* tempregion,
```

Comment:
In NSD it's a problem because no prefixing is done. simdzone uses the prefix `zone_` consistently for all exported symbols. So, while `_t` is technically reserved by the specification, I'd say chances of name conflicts are very very minimal, especially because it's common practice to use `_t`. The case is very different from using names starting with `_` or `__`.